compoments/report-repair/repair-description.js

```diff
@@ -136,6 +137,45 @@ const RepairDescription = ({handleChange, values}) => {
     return errorSummaryTextAndLocation;
   }
 
+  const ImageUploadRender = () => {
+    if (process.env.releaseVersion == 'mvp'){
+      return <div className={error.img ? 'govuk-form-group--error' : 'govuk-form-group'}>
```

Comment:
I'm getting loads of indentation issues in `ImageUploadRender`.

Reply:
Applied formatter. Could you please check to make sure I've resolved the indentation issues @agnieszkazbikowska-madetech